vtescrypt.csv

```diff
@@ -332,7 +332,7 @@
 "201669","Clara Hjortshøj","","Vampire","Tzimisce","","6","5","ani aus dom PRO","Anarch.","V5C:PTz1","","","Heather V. Kreiter"
 "200276","Clarissa Steinburgen","","Vampire","Nosferatu","","3","3","ani obf","Camarilla.","Anarchs:U2","","","Jim Nelson"
 "200277","Claudia","","Vampire","Lasombra","","5","2","obt","Sabbat: Black Hand.","DM:C/2","","","Nicolas ""Dimple"" Bigot"
-"200278","Claudio Severino","","Vampire","Tremere","","5","9","cel obf AUS DOM THA","Camarilla: Combat cards cost an opposing vampire an additional blood. During your unlock phase, burn 1 pool if there are no Gehenna cards in play.","Promo-20100510","","","Ginés Quiñonero-Santiago"
+"200278","Claudio Severino","","Vampire","Tremere","","5","9","cel obf AUS DOM THA","Camarilla: Combat cards cost an opposing vampire an additional blood. During your unlock phase, burn 1 pool if there are no Gehenna cards in play.","Promo-20100510","","","Ginés Quiñonero"
```

Comment:
Might wanna ask Ginés but I would have supposed, the other way around. Looks like Ginés Quiñonero-Santiago is used on the newest